home/components/AccountView.tsx

```diff
@@ -266,16 +242,17 @@ function ProfileProjectsSection({
   };
 
   const renderContents = () => {
-    if (!apps?.length) {
-      return <EmptyProfileProjectsNotice isOwnProfile={isOwnProfile} />;
+    const apps = data.account.byName.apps;
+    if (!apps.length) {
+      return <EmptyAccountProjectsNotice />;
     }
     const otherApps = takeRight(apps, Math.max(0, apps.length - MAX_APPS_TO_DISPLAY));
     return (
       <>
         {take(apps, MAX_APPS_TO_DISPLAY).map(renderApp)}
         <SeeAllProjectsButton
           apps={otherApps}
-          appCount={appCount - MAX_APPS_TO_DISPLAY}
```

Comment:
note that I need to make a backend change to support the `appCount` field on Account. This PR won't be landed until this is available in www prod and updated.

Reply:
https://github.com/expo/universe/pull/7241